### PR TITLE
Update site-wide-analysis.md

### PR DESCRIPTION
**Pull request by valdislav**
## Purpose of this pull request

Update documentation for Site-Wide Analysis Tool an agent. Fix systemd configuration.

## Affected DevDocs pages

[https://devdocs.magento.com/tools/site-wide-analysis.html](https://devdocs.magento.com/tools/site-wide-analysis.html)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@valdislav thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by yvoronoy** [comment]
@valdislav and @jeff-matthews Engineering CR Passed, we should be good to proceed and merge
Thank you!

File: src/tools/site-wide-analysis.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ We recommend configuring the agent to run as a service. If you have limited acce
 
 #### Service
 
-1. Create a systemd unit file (`/etc/systemd/system/scheduler.service`) with the following configuration.
+1. Create a systemd unit file (`/etc/systemd/system/scheduler.service`) with the following configuration. (Replace `<filesystemowner>` with unix user owner of /path/to/agent directory)
```
Comment (jeff-matthews) [inline src/tools/site-wide-analysis.md:231]:
```suggestion
1. Create a systemd unit file (`/etc/systemd/system/scheduler.service`) with the following configuration (replace `<filesystemowner>` with the Unix user that owns the directory where the agent is installed).
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

**Review by jeff-matthews — APPROVED** [review]

**Comment by jeff-matthews** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@jeff-matthews the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-valdislav-devdocs-9378`.

**Pull request merged**